packages/botonic-plugin-flow-builder/src/content-fields/whatsapp-button-list/flow-whatsapp-button-list-row.tsx

```diff
@@ -0,0 +1,26 @@
+import { ContentFieldsBase } from '../content-fields-base'
+import { HtWhatsappButtonListRow } from '../hubtype-fields'
+
+export class FlowWhatsappButtonListRow extends ContentFieldsBase {
+  public title = ''
+  public description = ''
+  public target?: string
+
+  static fromHubtypeCMS(
+    component: HtWhatsappButtonListRow,
+    locale: string
+  ): FlowWhatsappButtonListRow {
+    const newRow = new FlowWhatsappButtonListRow(component.id)
+    newRow.title = ContentFieldsBase.getTextByLocale(locale, component.text)
```

Comment:
nit: You can call another `static` method in the same class using `this` instead of the name of the class.
If you want to change it, change it in all the other classes calls also.

Reply:
applied on previous PR